### ISSUE TITLE
hwdb POINTINGSTICK_CONST_ACCEL for trackpoint is ignored for other than scrolling (Thinkpad Yoga 260)

### ISSUE THREAD

**Issue by ghost** [systemd/systemd#5902]
### Submission type

  - [X] Bug report
  - [ ] Request for enhancement (RFE)

*NOTE: Do not submit anything other than bug reports or RFEs via the issue tracker!*

### systemd version the issue has been seen with
```
systemd 232
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN
```

### Used distribution

Ubuntu 17.04

### In case of bug report: Expected behaviour you didn't see

I added this hwdb file, updated the hwdb with the command as specified in 70-pointingstick.hwdb and rebooted:
```
e@e:~$ sudo cat /etc/udev/hwdb.d/71-pointingstick-local.hwdb 

evdev:name:* Elantech TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadYoga260:*
 POINTINGSTICK_CONST_ACCEL=10.0

e@e:~$
```
After rebooting, I would have expected the mouse to be super fast when moved with the trackpoint.

### In case of bug report: Unexpected behaviour you saw

Scrolling with the trackpoint's middle mouse button is now super fast (almost uncontrollably so), but the mouse cursor when moved regularly has the same slow borderline unbearable default speed that it had before.

### In case of bug report: Steps to reproduce the problem

1. Use a Thinkpad Yoga 260 & Ubuntu 17.04 with GNOME 3 installed as provided by the distribution packages
2. Add this to `/etc/udev/hwdb.d/71-pointingstick-local.hwdb`:
```
evdev:name:* Elantech TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadYoga260:*
 POINTINGSTICK_CONST_ACCEL=10.0
```
3. Reload udev devices, update hwdb with `udevadm hwdb --update` and reboot
4. Launch GNOME 3 Xorg session
4. Move mouse around with trackpoint

**Comment by osteenbergen** [comment]
I have the same issue on OpenSUSE Tumbleweed and a Thinkpad 13.

The Elantech TrackPoint is slow, needs too much force to move the cursor. I have tried changing the xinput props, but there aren't many that improve usability (except Coordinate Transformation Matrix).

Today I found the hwdb setting to improve the acceleration, however as JonasT mentions it only works on the scrolling speed not the pointer speed.

My hwdb file read:
```
evdev:name:*Elantech TrackPoint:dmi:bvn*:bvr*:bd*:svn*:pn*:pvr*:*
 POINTINGSTICK_CONST_ACCEL=5.0
```

And I update the rules using
```
udevadm hwdb -u
udevadm trigger /dev/input/event5
```
